app/src/vuex/modules/session.js

```diff
@@ -353,6 +358,45 @@ export default ({ apollo }) => {
     setNotificationAvailable(store, { notificationAvailable }) {
       state.notificationAvailable = notificationAvailable
     },
+    async getAllUsedAddresses(store) {
```

Comment:
let'S move this to the user menu component and make it a computed

Reply:
did we make a decision here?

Reply:
Done